readthedocs/projects/static/projects/css/import.css

```diff
@@ -49,12 +49,14 @@ div.project-import-remote li.remote-repo ul.remote-repo-menu {
   top: 10px;
   right: 10px;
 }
-div.project-import-remote form.import-connect {
-  overflow: auto;
-  display: inline;
+div.project-import-remote div.project-import-providers ul {
+  overflow: hidden;
+}
+div.project-import-remote div.project-import-providers ul li {
+  float: left;
 }
-div.project-import-remote form.import-connect button {
-  display: inline;
+div.project-import-remote a.socialaccount-provider {
+  display: block;
```

Comment:
These changes look to have some negative side effect on spacing and alignment. What prompted these changes? If we have to alter layout on one of the forms, that might be better.

Reply:
"I don't know CSS", so I don't have _good reasons_. I touched it a little, googled something and it looked it worked so I thought that I could propose it.

I'm totally 👍 on remove this and change it as you like/prefer/wish/etc.

Reply:
Also, missed this the first time. This is actually editing a compiled css file, the source is at `readthedocs/projects/static-src/projects/css/import.less`. I'm going to be working to unify our styling, and to get rid of our `core.css` file soon.